app/src/main/java/org/thoughtcrime/securesms/database/LokiAPIDatabase.kt

```diff
@@ -179,7 +179,8 @@ class LokiAPIDatabase(context: Context, helper: SQLCipherOpenHelper) : Database(
                 val port = components.getOrNull(1)?.toIntOrNull() ?: return@mapNotNull null
                 val ed25519Key = components.getOrNull(2) ?: return@mapNotNull null
                 val x25519Key = components.getOrNull(3) ?: return@mapNotNull null
-                Snode(address, port, Snode.KeySet(ed25519Key, x25519Key))
+                val version = components.getOrNull(4) ?: "0.0.0"
```

Comment:
We use this "0.0.0" for unknown / null version number a bit - could make it into a constant to avoid fat-finger errors.